website/source/guides/identity/secure-intro.html.md

```diff
@@ -133,6 +137,37 @@ this model can be applied regardless of where the applications are running.
   - [Cubbyhole Response Wrapping](/guides/secret-mgmt/cubbyhole.html)
 
 
+### Vault Agent
+
+Vault agent allows the implementation of the [platform integration](#platform-integration)
```

Comment:
I would rephrase, something like "Vault agent automates the workflow of client login and token refresh, and can be used with either platform integration or trusted orchestrator approaches."

Reply:
Thank you!  Incorporated in [147540b](https://github.com/hashicorp/vault/pull/5323/commits/147540b7cda18f2ee14aa4ca3b758e44b6e4d8d8)